app/Extensions/PaymentGateways/MercadoPago/MercadoPagoSettings.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Extensions\PaymentGateways\MercadoPago;
+
+use Spatie\LaravelSettings\Settings;
+
+class MercadoPagoSettings extends Settings
+{
+
+    public bool $enabled = false;
+    public ?string $access_token;
```

Comment:
When you add the access_token as encrypted in the DB Migration it needs to be markes es encrypted in the Settings Class too.
https://github.com/spatie/laravel-settings?tab=readme-ov-file#encrypting-properties

Reply:
This is still open*

Reply:
@drylian